pom.xml

```diff
@@ -89,6 +93,34 @@
             </plugins>
         </pluginManagement>
     </build>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git://github.com/stargate/stargate.git</connection>
+        <developerConnection>scm:git:git@github.com:stargate/stargate.git</developerConnection>
+        <url>http://github.com/stargate/stargate/tree/master</url>
```

Comment:
Why not just the base URL of `http://github.com/stargate/stargate`? ... just curious.

Reply:
Because I copy pasted the example from sonatype :)